src/util/mod.rs

```diff
@@ -37,3 +33,27 @@ pub use crate::error::Error;
 
 #[deprecated(since = "0.30.0", note = "Please use crate::util::uint::BitArray")]
 pub use crate::util::uint::BitArray;
+
+/// The bip32 module can now be accessed from the crate root.
+#[deprecated(since = "0.30.0", note = "Please use crate::32")]
```

Comment:
In 871216fa4f1c22c5e1d76169ba662a6aa5c595c8:

`crate::32` should be `crate::bip32` in the deprecation message

Reply:
Oh not that super useful crate `crate::32`? :)